app/services/census_authorization_handler.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+# Checks the authorization against the census for Barcelona.
+class CensusAuthorizationHandler < Decidim::AuthorizationHandler
+  include ActionView::Helpers::SanitizeHelper
+
+  attribute :document_number, String
+  attribute :document_type, String
+  attribute :postal_code, String
+  attribute :date_of_birth, Date
```

Comment:
Do we have any legal age to participate in `decidim.barcelona`? In that case, we should implement it as a validation.